docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/dynamic-data-api-integration.md

```diff
@@ -136,6 +136,114 @@ class DynamicEntityBackendApiConfig extends SprykerDynamicEntityBackendApiConfig
 }
 ```
 
+### Dynamic Data import configuration
+
+4. It is also possible to import default configuration data, create configuration file:
```

Comment:
Phrasing is a bit off here. It's possible to import and create file? I guess you meant that it's possible to _install_ default configuration and for that you need to create configuration file